config/secrets.yml

```diff
@@ -14,10 +14,10 @@ default: &default
   artsy_application_id: <%= ENV['ARTSY_APPLICATION_ID'] %>
   artsy_internal_secret: <%= ENV['ARTSY_INTERNAL_SECRET'] %>
   artsy_application_secret: <%= ENV['ARTSY_APPLICATION_SECRET'] %>
-  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID']  || 'fake-id' %>
   aws_bucket: <%= ENV['AWS_BUCKET'] %>
-  aws_region: <%= ENV['AWS_REGION'] %>
-  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  aws_region: <%= ENV['AWS_REGION']  || 'fake-region' %>
+  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY']  || 'fake-key' %>
```

Comment:
Rather than adding fallbacks here, I think it might be nice to simply add these values to the `.env.test` file.